Packs/Zoom/Integrations/Zoom/Zoom.yml

```
@@ -1197,8 +1204,69 @@ script:
     - contextPath: Zoom.ChatMessage.at_items.start_position
       type: integer
       description: The start position of the mention.
-
-  dockerimage: demisto/pyjwt3:1.0.0.72374
+  - arguments:
+    - description: The name of the channel to archive. If not provided, the mirrored investigation channel is archived (if the channel exists).
+      name: channel
+    - description: The ID of the channel to archive. If not provided, the mirrored investigation channel is archived (if the channel exists).
+      name: channel_id
+    description: Delete a mirrored zoom channel.
+    name: close-channel
+  - name: send-notification
+    description: Send a message using a chatbot app.
+    arguments:
+    - name: message
+      description: the message.
+      required: true
+      type: string
+    - name: to
+      description: The user ID, JID or email to whom to send the message. You must provide this parameter or the channel_id parameter.
+      type: string
+    - name: channel_id
+      description: The ID or the JID of the channel to which to send the message. You must provide this parameter or the to parameter.
+      type: string
+    - name: channel
+      type: string
+      description: The channel name can be provided only if the channel is part of a mirroring incident, instead of the channel JID  .
+    - name: visible_to_user
+      description: The UserID that allows a Chatbot to send a message to a group channel when it has only one designated person in that group channel to see the message.
+      type: string
+    - name: zoom_ask
+      description: The message as a json for asking questions the user .
```

```suggestion
      description: The message as a JSON for asking questions to the user.
```
Is this really a boolean?  If so, the description is not clear.